Sources/Client.swift

```diff
@@ -45,14 +45,15 @@ open class Client {
         }
 
         self.configuration = configuration
-        self.spaceIdentifier = spaceIdentifier
+        self.spaceId = spaceId
     }
 
-    fileprivate func fetch<T: Decodable>(url: URL?, completion: @escaping (Result<T>) -> Void) -> URLSessionDataTask? {
+    // TODO: rename
```

Comment:
Do we keep todos in here, or should we have github issues about them. Also: How should it be renamed? Is :cow: okay?